### PR TITLE
feature/WDP190703-32/section_feedbac

### PR DESCRIPTION
**Pull request by kamil-polanski**
add section feedback

### CODE REVIEW

File: src/sass/components/_section--feed.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  i {
+    font-size: 22px;
+  }
+
+  img {
+    border-radius: 4px;
+    width: 60px;
+    height: 50px;
+    background-size: contain;
```
Comment (danielmark0116) [inline src/sass/components/_section--feed.scss]: fotka-miniaturka klienta wizualnie jest rozciagnieta

Comment (danielmark0116) [inline src/sass/components/_section--feed.scss]: Client pic is stretched :C

File: src/sass/components/_section--feed.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.section--feed {
+  p {
+    font-size: 14px;
+    line-height: 24px;
+    color: rgb(103, 103, 103);
```
Comment (matwjablonski) [inline src/sass/components/_section--feed.scss]: color from variable

File: src/sass/components/_section--feed.scss (hunk 1 above)
Comment (kamil-polanski) [reply src/sass/components/_section--feed.scss]: It shouldn't be stretched anymore.

File: src/partials/90_section-feed.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
+          </p>
+        </div>
+      </div>
+      <div class="row justify-content-center">
+        <div class="media justify-content-center">
+          <img src="./images/feed1.png" class="mr-3 mb-5" alt="client photo">
+          <div class="media-body">
+            <p class="mt-1 mb-0 client">John</p>
+            <p class="mt-0 mb-1">Furniture clietn</p>
```
[the remark this answers is not in the document]
Comment (kamil-polanski) [reply src/partials/90_section-feed.html]: It is good idea. I used margin bottom as u suggested,

File: src/sass/components/_section--feed.scss (hunk 2 above)
Comment (kamil-polanski) [reply src/sass/components/_section--feed.scss]: I added new variable with this colour.

**Pull request merged**